keyboards/crkbd/keymaps/mvi/config.h

```diff
@@ -0,0 +1,150 @@
+/*
+This is the c configuration file for the keymap
+
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2015 Jack Humbert
+Copyright 2021 Emmanuel Vill�ger
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* Select hand configuration */
+
+#define MASTER_LEFT
+// #define MASTER_RIGHT
+// #define EE_HANDS
+
+// #define SSD1306OLED
+#undef USE_I2C
+#undef SSD1306OLED
+
+#define USE_SERIAL_PD2
+
+#ifdef OLED_DRIVER_ENABLE
+   // Time unused before switching off oled screeen
+   #define OLED_SWITCHOFF 30000
+#endif
+
+// Configure tapping term for super alt/tab macro
+#define ALT_TAB_TERM 500
+
+// Enable rapid switch from tap to hold, disables double tap hold auto-repeat.
+//#define TAPPING_FORCE_HOLD
+
+// Configure the global tapping term (default: 200ms)
+#define TAPPING_TERM 300
+
+// Prevent normal rollover on alphas from accidentally triggering mods.
+#define IGNORE_MOD_TAP_INTERRUPT
+
+// Apply the modifier on keys that are tapped during a short hold of a modtap
+#define PERMISSIVE_HOLD
+
+//#define RETRO_TAPPING
+
+#ifdef RGBLIGHT_ENABLE
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
+>>>>>>> 626f577dd (changes according to PR, and unicode for uint16 keycodes)
+#    undef RGBLED_NUM
+#    define RGBLED_NUM 27
+#    define RGBLIGHT_ANIMATIONS
+#    define RGBLIGHT_SLEEP
+#    define RGBLIGHT_SPLIT
+#    define RGBLIGHT_LIMIT_VAL 120
+#    define RGBLIGHT_HUE_STEP 10
+#    define RGBLIGHT_SAT_STEP 17
+#    define RGBLIGHT_VAL_STEP 17
+<<<<<<< HEAD
```

Comment:
I think you might've had a problematic merge. Might want to go through your entire PR again.